packit/api.py

```diff
@@ -2401,17 +2400,17 @@ def run_osh_build(
             csmock_args = self.package_config.csmock_args
 
         if csmock_args:
-            cmd.append("--csmock-args=" + shlex.quote(csmock_args))
+            cmd.append("--csmock-args=" + csmock_args)
 
         osh_config = str(chroot)
```

Comment:
Why do we use the explicit `str(...)` conversion while assigning from `chroot` but not while assigning from `osh_options.config`?